ERCS/erc-7824.md

```diff
@@ -0,0 +1,146 @@
+---
+eip: 7824
+title: ERC for Exit Format in State Channels
+description: Outcomes and asset allocations definition in state channels using the Exit Format.
+author: State Channels (@statechannels), Layer-3 Foundation (@layer-3), Louis Bellet (@mod)
+discussions-to: https://ethereum-magicians.org/c/ercs/57
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-11-22
+---
+
+## Abstract
+
+This ERC defines a standard **Exit Format** for specifying outcomes and asset allocations in state channels. The Exit Format supports diverse token types (native tokens, ERC-20, and others) and accommodates both simple and advanced allocation mechanisms. The standard ensures interoperability and clarity in the distribution of locked assets upon channel finalization, particularly in protocols like Nitro.
```

Comment:
I'd like to see a brief definition of what an "exit format" is in your abstract, for anyone who isn't familiar with the term.